presto-benchmark/src/main/java/com/facebook/presto/benchmark/AbstractSimpleOperatorBenchmark.java

```diff
@@ -17,6 +17,7 @@
 import com.facebook.presto.operator.DriverContext;
 import com.facebook.presto.operator.DriverFactory;
 import com.facebook.presto.operator.OperatorFactory;
+import com.facebook.presto.operator.PagesIndex;
```

Comment:
Since we don't enable spill in production yet we will implement in-memory version first and support spill in a later PR.